app/javascript/components/student/solution-summary/Header.tsx

```diff
@@ -90,23 +110,31 @@ export const Header = ({
       )
     }
     case IterationStatus.NO_AUTOMATED_FEEDBACK:
+      if (exerciseType === 'tutorial') {
```

Comment:
@iHiD Is this correct? I show the same header regardless of the iteration status.

Reply:
Yes. It's ok for now, but it will need changing before launch. I have it as as TODO to work out with @taiyab .